src/Sylius/Bundle/WebBundle/Menu/FrontendMenuBuilder.php

```diff
@@ -260,8 +260,8 @@ public function createTaxonsMenu()
         foreach ($taxons as $taxon) {
             $child = $menu->addChild($taxon->getName(), $childOptions);
 
-            if ($taxon->hasPath()) {
-                $child->setLabelAttribute('data-image', $taxon->getPath());
+            if ($taxon->hasImages()) {
+                $child->setLabelAttribute('data-image', $taxon->getImages()->first()->getPath());
```

Comment:
`getImage()` :-1:
`getFirstImage()` (is it an implementation detail?) :+1:
 `getDefaultImage()` (allows for creating some prioritized list in the future) :+1:

We need to be explicit, having both `getImages()` and `getImage()` methods is quite weird.